platform/accessibility/common-issues/pdf/no-structural-tags.md

```diff
@@ -0,0 +1,101 @@
+    - [Adobe Acrobat Pro accessibility tools](#adobe-acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Statement of the issue
+
+- **Synopsis:** PDFs (portable document files) must have tagged headers, tables, lists, and paragraphs. Assistive technology can navigate to sections of a document using these tags. PDFs without tags require users to read the document instead of skimming for what they want.
+- **Severity:** Serious
+- **Deque University help link:** https://dequeuniversity.com/rules/comply-pdf/1.0/pdf-not-marked
+- **WCAG Success Criteria:** 
+  - [1.3.1 Info and Relationships](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
+  - [2.4.1 Bypass Blocks](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)
+
+## Definition of done
+
+- Ensure **Document is tagged PDF** is checked in the Adobe Acrobat accessibility checker options
+- There MUST be **0 untagged document violations** in the Adobe Acrobat [Accessibility Checker](https://www.adobe.com/accessibility/products/acrobat/using-acrobat-pro-accessibility-checker.html)
+- Consider exporting an accessibility report from Adobe Acrobat while running the Accessibility Checker
+
+## Tag your document
+
+Documents can be tagged in Microsoft Word as you create them. Use the built in tools to create headings, lists, and tables, then export your document as a tagged PDF.
+
+### Define headings in Microsoft Word
+
+Documents should have one **Title** and one **Heading 1**. **Heading 1** should be the main idea or message. Documents can have mutiple **Heading 2s** to communicate the most important supporting ideas. **Heading 3s** can be used for nested details.
+
+1. Select the **Home** tab
+2. Select or click and drag to highlight the text you want to convert to a heading
+3. Select the heading level you want
+4. Repeat steps 1-3 for additional headings
+5. Confirm no accessibility violations using the [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+
+![Clicking the Heading 1 button will convert a line of normal text to a heading level one](https://user-images.githubusercontent.com/934879/90903853-25f99500-e394-11ea-8146-161b90e31ff3.jpg)
+
+---
+
+### Define lists in Microsoft Word
+
+1. Select the **Home** tab
+2. Select or click and drag to highlight the text you want to convert to a list
+3. Select the Unordered List or Ordered List button to choose the type of list you want
+4. Repeat steps 1-3 for additional lists
+5. Confirm no accessibility violations using the [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+
+![Clicking the ordered list or unordered list button will convert normal text to a list](https://user-images.githubusercontent.com/934879/90903746-006c8b80-e394-11ea-84c7-2f45c1e1d6de.jpg)
+
+---
+
+### Export a tagged PDF
+
+1. Select the **Acrobat** tab
+2. Select **Create PDF**
+3. Name your document memorably
```

Comment:
Consider changing step 3 to:
In the **Save As** field, enter a file name for your document.